spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkMetadataColumns.java

```diff
@@ -311,6 +317,22 @@ public void testConflictingColumns() {
         sql("SELECT _spec_id, _partition, _renamed_spec_id FROM %s", TABLE_NAME));
   }
 
+  @TestTemplate
+  public void testRowLineageColumnsResolvedInV3OrHigher() {
+    if (formatVersion >= 3) {
+      assertEquals(
+          "Rows must match",
+          ImmutableList.of(),
+          sql("SELECT _row_id, _last_updated_sequence_number, id FROM %s", TABLE_NAME));
```

Comment:
I intentionally chose an empty table, since the goal was to ensure that the columns could be resolved in V3 or higher. If we insert data , the assertions would later have to change when we add inheritance , which seemed like an unnecessary burden considering the scope of this test.

Reply:
Seems worth a comment to explain?